ui/shared/status/StatusChatInput.qml

```diff
@@ -555,13 +555,13 @@ Rectangle {
             anchors.rightMargin: 0
             ScrollBar.horizontal.policy: ScrollBar.AlwaysOff
             height: {
-                if (messageInputField.height <= messageInput.defaultInputFieldHeight) {
+                if (messageInputField.implicitHeight <= messageInput.defaultInputFieldHeight) {
                     return messageInput.defaultInputFieldHeight
                 }
-                if (messageInputField.height >= messageInput.maxInputFieldHeight) {
+                if (messageInputField.implicitHeight >= messageInput.maxInputFieldHeight) {
                     return messageInput.maxInputFieldHeight
                 }
-                return messageInputField.height
```

Comment:
I'm actually a bit mad that it was this simple since I spent about an hour last time trying to figure it out and it didn't work